### PR TITLE
Rename package github.com/sylabs/singularity to github.com/hpcng/singularity

### PR DESCRIPTION
**Pull request by cclerget**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ to build Singularity from source:
 ```
 $ mkdir -p ${GOPATH}/src/github.com/sylabs && \
```
Comment (DrDaveD) [inline INSTALL.md]: This should be  hpcng, not sylabs.

File: go.mod [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module github.com/sylabs/singularity
+module github.com/hpcng/singularity
 
 go 1.13
```
Comment (DrDaveD) [inline go.mod:3]: How about the copy of golang.org/x/crypto in github.com/sylabs/golang-x-crypto, should we move that too?

**Review by DrDaveD — CHANGES_REQUESTED** [review]
A little more is needed in INSTALL.md, and perhaps the golang-x-crypto should be coming from hpcng.

File: INSTALL.md (hunk 1 above)
Comment (cclerget) [reply INSTALL.md]: fixed

File: go.mod (hunk 2 above)
Comment (cclerget) [reply go.mod:3]: Yes, change applied

**Review by DrDaveD — APPROVED** [review]
Looks good now

**Review by ikaneshiro — APPROVED** [review]

**Review by kmuriki — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**